Dockerfile

```diff
@@ -46,7 +46,8 @@ RUN svn co https://github.com/docker/docker/branches/$ENGINE_BRANCH/docs/extend
  && jekyll build -s allv -d allvbuild \
  && rm -rf allvbuild/apidocs/layouts \
  && find allvbuild -type f -name '*.html' -print0 | xargs -0 sed -i 's#href="https://docs.docker.com/#href="/#g' \
- && rm -rf allv
+ && rm -rf allv \
+ && apt-get update \
+ && apt-get -y install nginx
```

Comment:
I suggest to;

- move installation of nginx to a separate `RUN`, as it's not related to the generation of the archives. The installation step can be performed _before_ generation of the archives
- to stick close to the official nginx image, which is also based on `debian:jessie`, we can copy the same approach as that image; https://github.com/nginxinc/docker-nginx/blob/e950fa7dfcee74933b1248a7fe345bdbc176fffb/mainline/jessie/Dockerfile#L5-L23
- some of those modules are not needed, so we can omit those, so it'll be something like;

```Dockerfile
ENV NGINX_VERSION 1.11.9-1~jessie

RUN apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62 \
	&& echo "deb http://nginx.org/packages/mainline/debian/ jessie nginx" >> /etc/apt/sources.list \
	&& apt-get update \
	&& apt-get install --no-install-recommends --no-install-suggests -y \
						ca-certificates \
						nginx=${NGINX_VERSION} \
	&& rm -rf /var/lib/apt/lists/*

# forward request and error logs to docker log collector
RUN ln -sf /dev/stdout /var/log/nginx/access.log \
	&& ln -sf /dev/stderr /var/log/nginx/error.log
```